index.md

```diff
@@ -215,14 +215,14 @@ This is an overview of all Android versions and their corresponding identifiers
     <td><code>JELLYBEAN</code></td>
   </tr>
   <tr>
-    <td>4.0.3 – 4.0.4</td>
```

Comment:
I think it was better before. More space, but more explicit and therefore easier to read and understand.